pkg/controller/nodenetworkconfigurationpolicy/conditions/manager.go

```diff
@@ -16,12 +17,12 @@ import (
 
 type Manager struct {
 	client   client.Client
-	policy   nmstatev1alpha1.NodeNetworkConfigurationPolicy
+	policy   *nmstatev1alpha1.NodeNetworkConfigurationPolicy
 	nodeName string
 	logger   logr.Logger
 }
 
-func NewManager(client client.Client, nodeName string, policy nmstatev1alpha1.NodeNetworkConfigurationPolicy) Manager {
+func NewManager(client client.Client, nodeName string, policy *nmstatev1alpha1.NodeNetworkConfigurationPolicy) Manager {
```

Comment:
Only now I see how vague it is. we should make clear that this manages conditions in the Enactment (therefore it takes nodeName as an argument). Could you rename it to NewEnactmentConditionsManager? And the strcuture as well? Or maybe to follow the schema from previous PR, let's call the structure `EnactmentConditions`.

Reply:
Well we are updating policy conditions too, but maybe that was not a good idea, let's talk about this one.